appservice/src/tree/AppSettingsTreeItem.ts

```diff
@@ -10,12 +10,17 @@ import { AppSettingTreeItem } from './AppSettingTreeItem';
 import { getThemedIconPath } from './IconPath';
 import { ISiteTreeRoot } from './ISiteTreeRoot';
 
-export function validateAppSettingKey(settings: StringDictionary, newKey?: string, oldKey?: string): string | undefined {
-    newKey = newKey ? newKey.trim() : '';
-    oldKey = oldKey ? oldKey.trim().toLowerCase() : oldKey;
+export function validateAppSettingKey(settings: StringDictionary, newKey?: string, oldKey?: string, isLinux?: boolean): string | undefined {
```

Comment:
Set this as a default to false so it's explicit that we are assuming it's Windows.

`isLinux = false: boolean`

Add a clarifying comment as to WHY we'd rather assume it's Windows instead of Linux